src/main/java/com/solvd/app/jdbc/CustomerDAO.java

```diff
@@ -10,11 +10,13 @@
 import java.util.ArrayList;
 import java.util.List;
 
-public class CustomerDAO implements IBaseDAO<Customer> {
+public class CustomerDAO implements ICustomerDAO {
+
     private static final Logger LOGGER = LogManager.getLogger(CustomerDAO.class);
     private ConnectionPool connectionPool = ConnectionPool.getInstance();
     private PersonDAO personDAO = new PersonDAO();
 
+    @Override
     public void createEntity(Customer customer) {
```

Comment:
It will create a new customer and assign it with a new person_id, then this statement "customer.setCustomerID(newKey.getInt(1));" will retrieve the new key from the DB and set it to this customer.